.github/workflows/build.yaml

```diff
@@ -100,19 +87,11 @@ jobs:
         echo "::set-output name=artifacts-sha256::$DIGEST"
     # For now only generate artifacts for the specified OS and Python version in env variables.
     # Currently reusable workflows do not support setting strategy property from the caller workflow.
-    - name: Upload the package artifact for debugging
-      if: inputs.is_release_build == false && matrix.python == '3.10' && matrix.os == 'ubuntu-latest'
+    - name: Upload the package artifact for debugging and release
+      if: matrix.python == '3.10' && matrix.os == 'ubuntu-latest'
```

Comment:
Should this be `ARTIFACT_PYTHON` and `ARTIFACT_OS`?

Reply:
> 

:facepalm:

Reply:
See the change [here](https://github.com/jenstroeger/python-package-template/compare/14c9c101ba025e721d1e6d49d4d885fe3d4216a3..832d493422f1f2da25ed0f1817ded3befd76990f).